machines/flush-cache.js

```diff
@@ -42,16 +42,6 @@ module.exports = {
       }
     },
 
-    failed: {
```

Comment:
I've checked both Redis (http://redis.io/commands/FLUSHALL) and Memcached. Redis is very straightforward, saying 
> This command never fails.
In the case of Memcached, as you may have seen it just says
> This command does not pause the server, as it returns immediately
We can keep it, I kind of imagine someone implementing a cache to disk or other db, it might fail.